src/config/config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do not store such sensitive data in source code - it is unsecure. Consider using environmental variables for that (dotenv dependency is will help with env variables import)

Reply:
Done